app/Scripts/Stats.gd

```diff
@@ -1,27 +1,58 @@
 extends Node
-signal healthChange
-signal staminaChange
-var Health setget ChangedHealth
-var staminaForDash = 10
-var MaxHealth = 100
+#_____________________#
+
+signal Death
+signal healthChange(value)
+signal staminaChange(value)
+signal boidsChange(value)
+#_____________________#
+
+var BoidsCollectedNum setget BoidsChanged
 var Stamina setget ChangedStamina
+var Health = 10 setget ChangedHealth
+var FollowingBoids = []
+
```

Comment:
can we try to follow the gdscript style guide? there's no real private/public in godot and also it's kinda confusing to know what's a var or func or class without following consistent naming